test/unittest/models/unittest_models.cpp

```diff
@@ -882,6 +932,10 @@ GTEST_PARAMETER_TEST(
                  "transformer_pseudo_bool_attn_mask", ModelTestOption::ALL_V2),
     mkModelIniTc(fc_relu_decay, DIM_UNUSED, NOT_USED_,
                  ModelTestOption::COMPARE_V2),
+    mkModelTc_V2(non_trainable_fc_idx1, "non_trainable_fc_idx1",
```

Comment:
How about makeing names simliar with others ?  "make...."

Reply:
Thank you for your good point. I will correct it!